crates/store/src/state.rs

```diff
@@ -87,6 +94,21 @@ struct InnerState {
     account_tree: SimpleSmt<ACCOUNT_TREE_DEPTH>,
 }
 
+/// Data required to verify a block's inclusion proof.
+pub struct BlockInclusionProof {
+    pub block_header: BlockHeader,
+    pub mmr_path: MerklePath,
+}
+
+impl From<BlockInclusionProof> for miden_node_proto::generated::block::BlockInclusionProof {
```

Comment:
Following one of the above comments: I think it would be a bit easier to read this line if it was:
```rust
impl From<BlockInclusionProof> for BlockInclusionProofProto
```
or
```rust
impl From<BlockInclusionProof> for proto::BlockInclusionProof
```

Reply:
Ah yeah my `grep` missed this one. Thanks.